packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -1,35 +1,21 @@
-import React, { useReducer, useState, useRef, useEffect } from 'react';
-import { includes, isEmpty, toString, isEqual, intersectionWith } from 'lodash';
+import React, { useReducer, useState, useEffect } from 'react';
+import { includes, toString, isEqual, intersectionWith } from 'lodash';
 import { useHistory, useLocation } from 'react-router-dom';
+import { useIsMounted } from '@buffetjs/hooks';
 import { Header } from '@buffetjs/custom';
-import { useDebounce } from '@buffetjs/hooks';
 import {
-  HeaderSearch,
-  PageFooter,
   PopUpWarning,
-  LoadingIndicatorPage,
+  LoadingIndicator,
   useGlobalContext,
   generateFiltersFromSearch,
   generateSearchFromFilters,
   request,
   useQuery,
 } from 'strapi-helper-plugin';
-import {
-  formatFileForEditing,
-  getRequestUrl,
-  getTrad,
-  generatePageFromStart,
-  generateStartFromPage,
-  getFileModelTimestamps,
-} from '../../utils';
+import { formatFileForEditing, getRequestUrl, getTrad, getFileModelTimestamps } from '../../utils';
 import Container from '../../components/Container';
-import ControlsWrapper from '../../components/ControlsWrapper';
+import HomePageContent from '../../components/HomePageContent';
```

Comment:
In this file there's an issue with the header displayed count. In the `headerProps` object we should pass it `dataCount` instead of the `data`. You will need to update the `getHeaderLabel` utility so it takes an `Number` instead of an `Array` as argument.

Reply:
Are you sure we want to pass the data length? 
Because if we use it and we have 25 items but with a limit of 10, the number displayed will be 10 while we have 3 pages of items.

Reply:
Yes I am sure we need to display the total number of assets and not the total number of displayed ones

Reply:
I agree with it but it's not the data length that we should use so, because it can be the limit number and not the total of items.

Reply:
I understood, done!